e2e/03_new_wallet.test.js

```diff
@@ -0,0 +1,28 @@
+const { log, device, by, element } = require('detox');
```

Comment:
What's with the numerical prefixes on the file names?

Reply:
I don't really understand what you mean... I don't any numerical in `const { log, device, by, element } = require('detox');`... sorry in advance.

Reply:
I'm going to resolve this in order to merge it.